.travis.yml

```diff
@@ -13,5 +13,6 @@ install:
 script:
   - dnu restore Google.Common
   - dnu restore GoogleApiWrappers
+  - dnu restore Google.Apis.Common
   - dnu build */src/*
   - for project in `find . -name '*.Tests'`; do dnx -p $project test; done
```

Comment:
Do you mean between the restores and the build? I'm not sure that would work in YAML - wouldn't that break it as a list?